.github/workflows/build-v80.yml

```diff
@@ -0,0 +1,85 @@
+name: Build PHP V8.0
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
@grantlucas I will be doing that after the changes are merged into the main so it can build the initial versions. Otherwise, the moment it merges, it will not fire off any builds since nothing is getting changed in those paths.

Hence, I need to merge this PR. Once it builds the initial images and pushes them to the registry, I will then add the path as well in a follow up PR.